app/src/main/java/net/skyscanner/backpack/demo/compose/MapStory.kt

```diff
@@ -153,6 +159,78 @@ fun PointerMapMarkerStory(modifier: Modifier = Modifier) {
     }
 }
 
+@Composable
+@MapMarkersComponent
+@ComposeStory(kind = StoryKind.DemoOnly, name = "Location")
+fun LocationMapMarkerStory(modifier: Modifier = Modifier) {
+    GoogleMap(
+        modifier = modifier,
+        cameraPositionState = rememberCameraPositionState { MapPosition },
+        mapColorScheme = ComposeMapColorScheme.FOLLOW_SYSTEM,
+    ) {
+        MarkerPositions.forEachIndexed { index, latLng ->
+            BpkLocationMapMarker(
+                title = stringArrayResource(R.array.map_marker_prices)[index],
+                state = rememberMarkerState(position = latLng),
+            )
+        }
+    }
+}
+
+@Composable
+@MapMarkersComponent
+@ComposeStory(kind = StoryKind.DemoOnly, name = "POI")
+fun POIMapMarkerStory(modifier: Modifier = Modifier) {
```

Comment:
we generally name things `Poi` rather than all caps

Reply:
Because of the consideration of following the content of the design, so I also named it haha